crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -169,23 +212,31 @@ mod tests {
 
     #[tokio::test]
     async fn successful() {
-        let version = format!("{RPC_VERSION_YEAR}.{RPC_VERSION_MONTH}");
+        let version = format!(
+            "{}.{}",
+            env!("CARGO_PKG_VERSION_MAJOR"),
+            env!("CARGO_PKG_VERSION_MINOR")
+        );
+
```

Comment:
Construct this from the fields in `Version::for_test` instead of the `env!` variables, to avoid a silent dependency in this test.